docs/security.html

```diff
@@ -216,8 +216,9 @@
     };</pre>
 
         </li>
-        <li>Pass the name of the JAAS file as a JVM parameter to each Kafka broker:
+        <li>Pass the JAAS and krb5 file locations as JVM parameters to each Kafka broker. If no krb5 path is provided the JVM will use the system default for your OS:
```

Comment:
I think I'd say pass the JAAS and _optionally_ krb5 file locations or something like that. And maybe link to the Oracle docs instead of saying that the JVM will use the system default (it's a bit more complicated than that):

https://docs.oracle.com/javase/8/docs/technotes/guides/security/jgss/tutorials/KerberosReq.html